### ISSUE TITLE
No transparency after last update

### ISSUE THREAD

**Issue by joaomatos02** [alacritty/alacritty#7969]
### System

OS:macOS
Version: alacritty 0.14.0-dev (7077a5f4)

### Problem
Opacity stopped working. Alacritty's background is black when opacity=1 and becomes grey with opacity=0 without turning transparent

**Comment by chrisduerr** [comment]
Have you confirmed downgrading to a21adf1a5073329501cca7dccd8a9f7d7baecc88 fixes the issue?

**Comment by joaomatos02** [comment]
Tested all versions and then messed up and gave you wrong info, sorry!
It stopped working in 48c088a. Last working commit in a77f77c.

**Comment by chrisduerr** [comment]
Which macOS version?

**Comment by gersonsosa** [comment]
I tested both with my config and #7965 doesn't seem to fix it, it's somewhat transparent but not as before.

This is at 235bfe7729bb94c30434eae361e9d4fed0ea382b
<img width="1582" alt="image" src="https://github.com/alacritty/alacritty/assets/4366859/1c77be81-4c49-4341-909e-827ee48e5abc">

This is last working commit for me a77f77c
<img width="1465" alt="image" src="https://github.com/alacritty/alacritty/assets/4366859/781d964d-b0e9-40be-8fcd-4567acfc7625">

Config on both:
```
...
[window]
decorations = "Buttonless"
#decorations = "Full"
option_as_alt = "Both"
opacity = 0.92
...
``` 
<details><summary>Full config</summary>
<p>
```
import = [
    # "~/.config/alacritty/colors/tokyo-night-storm.toml",
    # "~/.config/alacritty/colors/tokyo-night.toml",
    # "~/.config/alacritty/colors/catppuccin/catppuccin-latte.toml",
    # "~/.config/alacritty/colors/catppuccin/catppuccin-mocha.toml",
    "~/.config/alacritty/colors/catppuccin/catppuccin-frappe.toml",
    # "~/.config/alacritty/colors/catppuccin/catppuccin-machiatto.toml",
]

live_config_reload = true

#[shell]
#program = "/bin/zsh"

[bell]
animation = "EaseOutQuad"
color = "0xf7768e"
duration = 300

[colors]
draw_bold_text_with_bright_colors = true

[font]
size = 18.0

[font.normal]
family = "Maple Mono NF"

[hints]
alphabet = "jfkdls;ahgurieowpq"

[[hints.enabled]]
command = "open"
hyperlinks = true
post_processing = true
regex = "(ipfs:|ipns:|magnet:|mailto:|gemini:|gopher:|https:|http:|news:|file:|git:|ssh:|ftp:)[^\u0000-\u001F\u007F-<>\"\\s{-}\\^⟨⟩`]+"

[hints.enabled.binding]
key = "U"
mods = "Control|Shift"

[hints.enabled.mouse]
enabled = true
mods = "None"

[[keyboard.bindings]]
key = "Comma"
mods = "Command"

[keyboard.bindings.command]
args = ["-c", "open ~/.config/alacritty/alacritty.toml"]
program = "sh"

[[keyboard.bindings]]
action = "SpawnNewInstance"
key = "N"
mods = "Command"

[[keyboard.bindings]]
chars = " "
key = "Space"
mods = "Alt"

[[keyboard.bindings]]
chars = "\u0015"
key = "Back"
mods = "Super"

[[keyboard.bindings]]
chars = "\u001Bb"
key = "Left"
mods = "Alt"

[[keyboard.bindings]]
chars = "\u001Bf"
key = "Right"
mods = "Alt"

[[keyboard.bindings]]
chars = "\u001BOH"
key = "Left"
mode = "AppCursor"
mods = "Command"

[[keyboard.bindings]]
chars = "\u001BOF"
key = "Right"
mode = "AppCursor"
mods = "Command"

[[keyboard.bindings]]
action = "ToggleSimpleFullscreen"
key = "F"
mods = "Command|Control|Alt"

[window]
decorations = "Buttonless"
#decorations = "Full"
option_as_alt = "Both"
opacity = 0.92

[window.dimensions]
columns = 140
lines = 40
```
</p>
</details>

**Comment by chrisduerr** [comment]
@madsmtm @kchibisov Looks like a macOS winit regression?

**Comment by moritzdietz** [comment]
Note: https://github.com/alacritty/alacritty/issues/153#issuecomment-2131365935

I came here to voice issues with emoji rendering on Linux Mint

**Comment by nixpulvis** [comment]
https://github.com/rust-windowing/winit/pull/3722 should fix this.

**Comment by madsmtm** [comment]
I changed my mind, we're going to go through with https://github.com/rust-windowing/winit/pull/3722, and it will likely be included in Winit v0.30.2.
